iota/commands/extended/get_transaction_objects.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from typing import Iterable, List, Optional
+
+import filters as f
+
+from iota import Transaction, TransactionHash
+from iota.commands.core import GetTrytesCommand
+from iota.commands import FilterCommand, RequestFilter
+from iota.filters import Trytes
+
+__all__ = [
+    'GetTransactionObjectsCommand',
+]
+
+
+class GetTransactionObjectsCommand(FilterCommand):
+    """
+    Executes `GetTransactionObjectsCommand` command.
+
+    See :py:meth:`iota.api.StrictIota.get_transaction_objects`.
```

Comment:
```suggestion
    See :py:meth:`iota.api.Iota.get_transaction_objects`.
```

(`StrictIota` contains only commands exposed by the IRI node; original inspiration taken from [redis](https://github.com/andymccurdy/redis-py/blob/2.10.6/redis/client.py#L347-L356))

Reply:
Thanks for noticing!